ci-operator/step-registry/ocm/ci/fastforward-multiple/ocm-ci-fastforward-multiple-commands.sh

```
@@ -0,0 +1,102 @@
+    for version in ${DESTINATION_VERSIONS}; do
+      branch="${branch_prefix}-${version}"
+      echo "INFO: Fast-forwarding ${owner_repo} main to branch: ${branch}"
+      log_file="${ARTIFACT_DIR}/fastforward-${owner_repo//\//-}-${branch}.log"
+
+      REPO_OWNER=${owner} \
+        REPO_NAME=${repo} \
+        SOURCE_BRANCH=main \
+        DESTINATION_BRANCH=${branch} \
+        "${SCRIPT_DIR}/../fastforward/ocm-ci-fastforward-commands.sh" >"${log_file}" 2>&1 ||
+        {
+          local err=$?
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)